### PR TITLE
remove explanatory text, fix a few nits

### PR DESCRIPTION
**Pull request by timbray**
Implemented the changes as suggested in Issue #7

### CODE REVIEW

File: draft-bormann-jsonpath-iregexp.md [hunk 1]
```diff
@@ -93,6 +96,9 @@ This document includes rules for converting I-Regexps for use with several well-
 
 {::boilerplate bcp14-tagged}
 
+The grammatical rules in this document are to be interpreted as ABNF,
+as described in {{-abnf}}.
```
Comment (cabo) [inline draft-bormann-jsonpath-iregexp.md:100]: Actually, the current ABNF needs RFC 7405 (case-sensitive %s...) as well.

Comment (timbray) [reply draft-bormann-jsonpath-iregexp.md:100]: Heh, I had no idea 7405 even existed.  OK.

Comment (timbray) [reply draft-bormann-jsonpath-iregexp.md:100]: Having read 7405, it's a delta against 5234 so strictly I guess only 7405 need be referenced but it seems more polite to reference both.

Comment (cabo) [reply draft-bormann-jsonpath-iregexp.md:100]: Indeed, that's what I usually do.

**Pull request merged**